manifest/README.md

```diff
@@ -108,6 +113,7 @@ s3:
   minimumPriority: "warning"
   skipExistingOnStartup: true
   sources: []
+  channels: []
 
 reportFilter:
   namespaces:
```

Comment:
Maybe an example after this section would to know how to use the channels properties.

Reply:
For a detailed explaination I will use the documentation.